docs/configuration.md

```diff
@@ -16,6 +16,7 @@ var uiManager = new playkit.ui.UIManager(player, config);
   targetId: string,
   debugActions?: boolean, // optional
   forceTouchUI?: boolean, // optional
+  controlBarHoverTimeout?: number, // optional
```

Comment:
the config is for entire UI so let's find a more suitable name then `controlBarHoverTimeout`
Maybe `userHoverTimeout` or `userActionTimeout`

Reply:
`hoverTimeout` ?

Reply:
that was the original one that was never used internally, and I have no objection for it.